packages/waku/src/minimal/client.ts

```diff
@@ -309,11 +331,17 @@ export const Slot = ({
       elements[id],
     );
   };
+  if (unstable_renderPrev) {
+    if (!elementsPromise.prev) {
+      throw new Error('Missing prev elements');
+    }
+    return renderSlot(elementsPromise.prev);
+  }
```

Comment:
I'm not happy with this, and hope to redesign it after finishing our current transition.